module/TaskManagement/src/TaskManagement/Service/EventSourcingTaskService.php

```diff
@@ -90,4 +102,89 @@ public function getEventManager()
 		}
 		return $this->events;
 	}
+	
+	public function getAcceptedTaskIdsToNotify(\DateInterval $timeboxForAcceptedTask){
+		
+		$builder = $this->entityManager->createQueryBuilder();
+		$query = $builder->select('t.id as TASK_ID')
+			->from(ReadModelTask::class, 't')			
+			->where('DATE_DIFF(CURRENT_DATE(), t.acceptedAt) = '.$timeboxForAcceptedTask->format('%d') . '-1')			
+			->andWhere('t.status = :taskStatus')			
+			->setParameter('taskStatus', Task::STATUS_ACCEPTED)			
+			->getQuery();			
+
+		return $query->getArrayResult();		
+	}
+	
+	
+	public function getAcceptedTaskIdsToClose(\DateInterval $timeboxForAcceptedTask){
+		
+		$builder = $this->entityManager->createQueryBuilder();
+		$query = $builder->select('t.id as TASK_ID')
+			->from(ReadModelTask::class, 't')			
+			->where('DATE_DIFF(CURRENT_DATE(), t.acceptedAt) >= :interval')
+			->andWhere('t.status = :taskStatus')			
+			->setParameter('interval', $timeboxForAcceptedTask->format('%d'))
+			->setParameter('taskStatus', Task::STATUS_ACCEPTED)
+			->getQuery();			
+			
+		return $query->getArrayResult();		
+
+	}
+	
+	public function notifyMembersForShareAssignment(Task $task, PhpRenderer $renderer, $taskMembersWithEmptyShares){
+

```

Comment:
Perché questo metodo sta in questo servizio?
Il nome non ne riflette la semantica